lib/topological_inventory/satellite/operations/source.rb

```diff
@@ -56,10 +57,7 @@ def availability_check_response(_msg_id, message_type, response)
           connected = response['result'] == 'ok' && response['fifi_status']
           status = connected ? STATUS_AVAILABLE : STATUS_UNAVAILABLE
 
-          unless available?(status)
-            logger.info("Source #{source_id} is unavailable. Result: #{response['result']}, FIFI status: #{response['fifi_status'] ? 'T' : 'F'}, Reason: #{response['message']}")
-          end
-
+          logger.info("Source #{source_id} is #{status}. Result: #{response['result']}, FIFI status: #{response['fifi_status'] ? 'T' : 'F'}, Reason: #{response['message']}")
```

Comment:
Can you transform this meessage to the common format like https://github.com/RedHatInsights/sources-api/pull/207